package testesigep.exception;

public class VendaNotFoundException extends RuntimeException {
    public VendaNotFoundException(Integer id) {
        super("Venda n√£o encontrado com o ID: " + id);
    }
}
